website/content/index.mdx

```diff
@@ -0,0 +1,201 @@
+  <div className='flex gap-3'>
+    <a href='https://circleci.com/gh/armadaproject/armada' target='_blank'>
+      <img
+        src='https://circleci.com/gh/helm/helm.svg?style=shield'
+        alt='CircleCI'
+      />
+    </a>
+    <a
+      href='https://goreportcard.com/report/github.com/armadaproject/armada'
+      target='_blank'
+    >
+      <img
+        src='https://goreportcard.com/badge/github.com/armadaproject/armada'
+        alt='Go Report Card'
+      />
+    </a>
+    <a
+      href='https://artifacthub.io/packages/helm/gresearch/armada'
+      title='Go to Artifact Hub'
+      target='_blank'
+    >
+      <img
+        src='https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/armada'
+        alt='Artifact Hub'
+      />
+    </a>
+    <a
+      href='https://insights.lfx.linuxfoundation.org/foundation/cncf/overview?project=armada'
+      target='_blank'
+      title='Click to view project insights and health checks'
+    >
+      <img
+        src='https://img.shields.io/badge/LFX%20Insights-0094FF'
+        alt='LFX Insights Dashboard'
+      />
+    </a>
+  </div>
+</div>
+
+## What is Armada?
+
+Armada is a multi-Kubernetes cluster batch job meta-scheduler designed to handle massive-scale workloads. Built on top of Kubernetes, Armada enables organizations to distribute millions of batch jobs per day across tens of thousands of nodes spanning multiple clusters, making it an ideal solution for high-throughput computational workloads.
+
+Armada serves as middleware that transforms Kubernetes into a powerful batch processing platform while maintaining compatibility with service workloads. It addresses the fundamental limitations of running batch workloads at scale on Kubernetes by providing:
+
+- **Multi-cluster orchestration**: Schedule jobs across many Kubernetes clusters seamlessly
+- **High-throughput queueing**: Handle millions of queued jobs
+- **Advanced batch scheduling**: Fair queuing, gang scheduling, preemption, and resource limits
+- **Enterprise-grade reliability**: Secure, highly available components designed for production use
+
+As a [CNCF Sandbox project](https://www.cncf.io/), Armada is actively maintained and used in production environments, including at [G-Research](https://www.gresearch.com/) where it processes millions of jobs daily.
+
+## Why Use Armada?
+
+### Kubernetes Limitations for Batch Workloads
+
+Traditional Kubernetes faces several challenges when running batch workloads at scale:
+
+1. **Single Cluster Scaling Limits**: Scaling a single Kubernetes cluster beyond a certain size is [challenging](https://openai.com/blog/scaling-kubernetes-to-7500-nodes/), typically maxing out around 5,000-15,000 nodes depending on configuration.
+
+2. **Storage Backend Constraints**: Achieving very high throughput using etcd, Kubernetes' in-cluster storage backend, is [challenging](https://etcd.io/docs/v3.5/op-guide/performance/) and can become a bottleneck for job queuing.
```

Comment:
"challenging" used and highlighted in two sequential items. it looks a little strange rendered.